tests/invalid_models_tests/test_models.py

```diff
@@ -436,7 +436,7 @@ class Model(models.Model):
             fk_2 = models.ForeignKey(Target, models.CASCADE, related_name="target_2")
 
             class Meta:
-                constraints = [
+                indexes = [
```

Comment:
This one made me wonder if we should have a check to ensure members of `Meta.constraints` are instances of `BaseConstraint`.

Reply:
I think it will fail anyway during migrations.

Reply:
True but that's the case for any other problem we added a system check for.